lab-catherine/__test__/greet.test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const greet = require('../lib/greet');
+
+describe('greet.test.js', () => {
+  describe('greet.null', () => {
+    test('testing to return null for non string values', () => {
+      expect(greet.hello(null)).toBeNull();
```

Comment:
you should also be inputting an empty string, and returning null, and also testing for a numeric input and returning null